archibishop/github-connect
# #156004609 Pass data to detail activity

#### What does this PR do?
Passes data to detail activity when it is launched.
#### Description of Task to be completed?
Passes data the detail activity when launched and populates the different widgets with data on that activity.
#### How should this be manually tested?
- Clone the repository.
- Run the application.
- Click on the on an item in the recycler view and the detail activity should be launched with the information of the item that was clicked.
#### What are the relevant pivotal tracker stories?
[#156004609](https://www.pivotaltracker.com/story/show/156004609)
#### Screenshots (if appropriate)
![passdata](https://user-images.githubusercontent.com/39958516/46269755-c6016780-c54b-11e8-8c81-83d45c07636e.gif)


<img width="277" alt="screen shot 2018-09-28 at 11 26 15" src="https://user-images.githubusercontent.com/39958516/46198910-3cb51f80-c316-11e8-822a-3d7bf4051d9f.png"><img width="283" alt="screen shot 2018-09-28 at 11 30 24" src="https://user-images.githubusercontent.com/39958516/46269797-0bbe3000-c54c-11e8-8164-b7e291c433bc.png">
